scripts/navbar_menu.js

```diff
@@ -46,6 +43,13 @@ function hideSiteSearch() {
 }
 
 window.addEventListener('DOMContentLoaded', function() {
+    const menuSelectors = document.getElementsByClassName('menu-selector');
+    for(const menuSelector of menuSelectors) {
```

Comment:
```suggestion
    for (const menuSelector of menuSelectors) {
```

Reply:
we have a linter which should enforce some of these suggestions I believe:
https://github.com/DistributedProofreaders/dproofreaders/blob/master/.eslintrc.json
If not, we can definitely codify our desired format.

Reply:
It appears to have [passed the eslint linter](https://github.com/DistributedProofreaders/dproofreaders/actions/runs/10391438980/job/28774349190?pr=1300)? Wonder if something's not configured quite right.

Reply:
Let me investigate. We may just not have too many format rules configured.